editor/src/components/canvas/canvas-strategies/strategies/set-grid-gap-strategy.tsx

```diff
@@ -171,30 +171,33 @@ export const setGridGapStrategy: CanvasStrategyFactory = (
         axis === 'row' ? updatedGridGapMeasurement.row : updatedGridGapMeasurement.column
 
       if (shouldTearOffGapByAxis) {
-        return strategyApplicationResult([
-          deleteProperties('always', selectedElement, [axisStyleProp]),
-        ])
+        return strategyApplicationResult(
+          [deleteProperties('always', selectedElement, [axisStyleProp])],
+          'rerender-all-elements',
```

Comment:
possibly one to fix here too

Reply:
Added comment about follow up in 38ec355d5.